lib/can/include/CANBus.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include "hw_bridge.h"
+
+class CANBus : public CAN {
```

Comment:
i needed this class to wrap CAN bc we need to hide CAN::filter without changing mbed. this also helps us customize CAN as we need it in the future